pom.xml

```diff
@@ -4,7 +4,7 @@
     <parent>
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>plugin</artifactId>
-        <version>1.401</version>
+        <version>2.21</version>
     </parent>
 
     <artifactId>versioncolumn</artifactId>
```

Comment:
Or maybe just `System Node Monitors Plugin`, `Advanced Node Monitors Plugin` or whatever. The description should help with details